omod/src/main/java/org/openmrs/module/cohort/web/resource/CohortMemberRequestResource.java

```diff
@@ -128,6 +136,25 @@ public void purge(CohortMember cohortMember, RequestContext context) throws Resp
 		throw new UnsupportedOperationException();
 	}
 
+	@PropertySetter("attributes")
+	public static void setAttributes(CohortMember cohortMember, Set<CohortMemberAttribute> attributes) {
+		for (CohortMemberAttribute attribute : attributes) {
+			attribute.setOwner(cohortMember);
+		}
+		cohortMember.setAttributes(attributes);
+	}
+
+	@PropertyGetter("display")
+	public String getDisplayString(CohortMember cohortMember) {
+		Patient patient = cohortMember.getPatient();
+		if (patient != null) {
+			PatientIdentifier identifier = patient.getPatientIdentifier();
+			return identifier + "-" + patient.getGivenName() + " " + patient.getMiddleName() + " " + patient.getFamilyName();
```

Comment:
Might be better as:

```suggestion
			return identifier + "-" + patient.getPersonName().getFullName();
```

This allows us to take advantage of system-specific name formatting (people do weird things with names)